landing.js

```diff
@@ -1,36 +1,59 @@
-$(function () {
-  initAutoComplete($("#autocomplete"), $("#goingnext"));
-  initAutoComplete($("#autocomplete_2"), $("#goingnext-2"));
-  initAutoComplete($("#autocomplete_3"), $("#goingnext-3"));
-  storeDataForProductPitchABTesting();
-});
-
 // used by the marketing team for analytics and AB testing, to track which page,
 // ad or marketing pitch was presented to the seller initially
 function storeDataForProductPitchABTesting() {
   const path = location.pathname.toLowerCase();
 
   const productPitches = [
-    [/\/(lp\/|)max($|\/.*)/, "MAX"],
-    [/\/(lp\/|)cover($|\/.*)/, "COVER"],
-    [/\/(lp\/|)vendre(-|\/).+/, "COVER"],
-    [/\/.*/, "NONE"],
+    [/\/(lp\/|)max($|\/.*)/, 'MAX'],
+    [/\/(lp\/|)cover($|\/.*)/, 'COVER'],
+    [/\/(lp\/|)vendre(-|\/).+/, 'COVER'],
+    [/\/.*/, 'NONE'],
   ];
-  localStorage.setItem("productPitch", productPitches.find((v) => v[0].test(path))[1]);
+  localStorage.setItem('productPitch', productPitches.find((v) => v[0].test(path))[1]);
 }
 
 let showError = true;
 function displayError() {
   if (showError) {
     showError = false;
-    alert("Merci de préciser votre numéro de rue.");
+    alert('Merci de préciser votre numéro de rue.');
     setTimeout(() => (showError = true), 1000);
   }
 }
 
-function useFirstPrediction($input) {
-  const value = $input.val();
-  if (value === "") {
+function createCookieAndRedirect(place) {
+  const componentForm = {
+    street_number: 'short_name',
+    route: 'long_name',
+    locality: 'long_name',
+    administrative_area_level_1: 'short_name',
+    country: 'long_name',
+    postal_code: 'short_name',
+  };
+
+  // for some reason, the first item it ine address_components object should be the street number, if it's not, we manually set the cookie for it
```

Comment:
`the first item it ine address_components`

did you mean `the first item in address_components`?

Reply:
Fixed